app/presenters/default_deadline_date_time.rb

```diff
@@ -0,0 +1,13 @@
+# used for computing the default deadline with an offset
+class DefaultDeadlineDateTime
+  attr_reader :dc_time
```

Comment:
I'd make this a method rather than an `attr_reader` -- think it's clearer that way. What do you think?